[Bug] UT test Failed  when BuildVolumesVolumeMountsAndPVCs_persistentVolume

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/doris-operator/issues?q=is%3Aissue) and found no similar issues.


### Version

25.5.3

### What's Wrong?

cd pkg/controller/sub_controller
go test -run TestDisaggregatedSubDefaultController_BuildVolumesVolumeMountsAndPVCs_persistentVolume



testcase will failed:

 --- FAIL: TestDisaggregatedSubDefaultController_BuildVolumesVolumeMountsAndPVCs_persistentVolume (0.00s)
    disaggregated_subcontroller_test.go:71: build be default volumes volumemounts and pvcs failed, the number is not right.
FAIL
exit status 1
FAIL	github.com/apache/doris-operator/pkg/controller/sub_controller	0.614s 



### What You Expected?

test case will not failed.

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/pkg/controller/sub_controller/disaggregated_subcontroller_test.go
+++ b/pkg/controller/sub_controller/disaggregated_subcontroller_test.go
@@ -67,7 +67,7 @@ func TestDisaggregatedSubDefaultController_BuildVolumesVolumeMountsAndPVCs_persi
     if len(fevs) != 2 || len(fevms) != 2 || len(fepvcs) != 2 {
         t.Errorf("build fe default volumes volumemounts and pvcs failed, the number is not right.")
     }
-    if len(bevs) !=3 || len(bevms) !=3 || len(bepvcs) != 3 {
+    if len(bevs) != 4 || len(bevms) != 4 || len(bepvcs) != 4 {
         t.Errorf("build be default volumes volumemounts and pvcs failed, the number is not right.")
     }
     if len(msvs) != 1 ||len(msvms) != 1 || len(mspvcs) != 1 {

